frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -471,8 +471,12 @@ class FragNavController internal constructor(builder: Builder, savedInstanceStat
             currentDialogFrag.dismiss()
             mCurrentDialogFrag = null
         } else {
-            val fragmentManager: FragmentManager = this.currentFrag?.childFragmentManager
-                    ?: this.fragmentManger
+            val fragmentManager: FragmentManager =
+                    if (this.currentFrag != null && this.currentFrag!!.isAdded) {
```

Comment:
Please extract this.currentFrag to a local variable then you'll not need to use !!

Reply:
oki doki ^__^